packages/terra-form-select/src/_Menu.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
This is for the intl prop. To remove this lint, we would have to make terra-base a required dependency of terra-form-select.

Reply:
Not even needed anymore. https://github.com/cerner/terra-core/pull/2163/commits/c80a4bd1bd23db67fc55748bccc0e1bde34e9ce3